### PR TITLE
[#155922719] user can see details for a specific asset via API 

### PR DESCRIPTION
**Pull request by iankigen**
#### What does this PR do?

Allow users to view a specific asset details by serial number via API 

#### Description of Task to be completed?

- add test for functionality
- add item viewset get object by serial number

#### How should this be manually tested?

`python manage.py test`
or 
endpoint `/api-auth/login` to login user
endpoint `/api/v1/items/{serial_number}` to view asset details

#### What are the relevant pivotal tracker stories?

[#155922719](https://www.pivotaltracker.com/n/projects/2146417/stories/155922719)

#### Screenshots 
view-by-serial

[delivers #155922719]

### CODE REVIEW

**Comment by iankigen** [comment]
@egichuri @alexkiura @HawiCaesar

File: api/views.py [hunk 1]
```diff
@@ -21,6 +22,10 @@ class ItemViewSet(ModelViewSet):
     http_method_names = ['get']
 
     def get_queryset(self):
-
         user = self.request.user
         return Item.objects.filter(assigned_to=user)
+
+    def get_object(self):
+        queryset = self.get_queryset()
```
Comment (alexkiura) [inline api/views.py]: Consider refactoring this to `queryset = Item.objects.filter(assigned_to=self.request.user)`. Otherwise looks good overall.

**Comment by HawiCaesar** [comment]
LGTM

**Comment by nelmandela** [comment]
@alexkiura done

**Pull request merged**